src/components/Badge/__snapshots__/Badge.spec.tsx.snap

```diff
@@ -130,8 +130,8 @@ exports[`Badge should have hover/active styles only when the onClick handler is
   border: 0;
 }
 
-<div
-  class="circuit-0 circuit-1"
+<button
```

Comment:
Why did this change?

Reply:
When the Badge is passed the `onClick` prop, it should render as a semantic `button` element. This wasn't the case before, so @hleote [fixed it](https://github.com/sumup-oss/circuit-ui/pull/624/files#diff-0443442098887b894fbbf2185b456647R151)